### DISCUSSION TITLE
use q key to any quotes in Match

### DISCUSSION THREAD

**Discussion by otavioschwanck** [helix-editor/helix#10395] [General]
`miq` would execute mi', mi", mi`, depending on where you are.  Super easier to select.

`maq` same thing.

**Comment by the-mikedavis** [comment]
I think it would be preferrable to fix https://github.com/helix-editor/helix/issues/4475 rather than add a textobject specific to quotes

**Comment by otavioschwanck** [comment]
> I think it would be preferrable to fix #4475 rather than add a textobject specific to quotes

sometimes the cursor are inside a string, but you want a quote.  In those case, is better to have the any quotes.
